src/Microsoft.ML.FastTree/doc.xml

```diff
@@ -157,28 +157,28 @@
       </summary>
       <remarks>
         In machine learning​ it is a pretty common and powerful approach to utilize the already trained model in the process of defining features.
-        <para>One such example would be the use of model's scores as features to downstream models. For example, we might run clustering on the original features, 
+        <para>One such example would be the use of model&apos;s scores as features to downstream models. For example, we might run clustering on the original features, 
```

Comment:
❔ Why is this required? It's awkward to read and seems like it would be easy to regress.